service/src/test/java/org/mskcc/smile/service/TempoServiceTest.java

```diff
@@ -276,6 +278,57 @@ public void testUpdateCohortCompleteData() throws Exception {
         Assertions.assertTrue(hasUpdates);
     }
 
+    @Test
+    public void testPopulatingTempoDataNoInitialRunDate() throws Exception {
+        // using a tumor sample to trigger population of tempo data
+        String igoId = "MOCKREQUEST1_B_3";
+        String requestId = "MOCKREQUEST1_B";
+        SmileSample sample = sampleService.getResearchSampleByRequestAndIgoId(requestId, igoId);
+
+        Tempo tempo = new Tempo();
+        tempo.setSmileSample(sample);
+        Tempo savedTempo = tempoService.saveTempoData(tempo);
+
+        Assertions.assertEquals("MSK Embargo", savedTempo.getAccessLevel());
+        Assertions.assertEquals("", savedTempo.getInitialPipelineRunDate());
+        Assertions.assertEquals("", savedTempo.getEmbargoDate());
+    }
+
+    @Test
+    public void testPopulatingTempoDataWithInitialRunDate() throws Exception {
+        // using a tumor sample to trigger population of tempo data
+        String igoId = "MOCKREQUEST1_B_1";
+        String requestId = "MOCKREQUEST1_B";
+        SmileSample sample = sampleService.getResearchSampleByRequestAndIgoId(requestId, igoId);
+
+        // save a cohort complete event to establish an initial pipeline run date
+        CohortCompleteJson ccJson = getCohortEventData("mockCohortCompleteCCSPPPQQQQ");
+        cohortCompleteService.saveCohort(new Cohort(ccJson), ccJson.getTumorNormalPairsAsSet());
+
+        Tempo tempo = new Tempo();
+        tempo.setSmileSample(sample);
+        Tempo savedTempo = tempoService.saveTempoData(tempo);
+
+        // assert that custodian information is set
+        Assertions.assertFalse(savedTempo.getCustodianInformation().isEmpty());
+
+        // assert that initial pipeline run date is set
+        Assertions.assertFalse(savedTempo.getInitialPipelineRunDate().isEmpty());
+
```

Comment:
For a little more clarity I would confirm that these fields are the expected values instead of just checking if they're not empty.